aidies-project/client/src/components/Navigation.js

```diff
@@ -1,18 +1,39 @@
 import React from 'react';
+import '../css/navigation.css';
+import logo from '../images/aidies-logo.svg';
 import { NavLink } from 'react-router-dom';
 import LogoutButton from './LogoutButton';
+import {Button} from '@material-ui/core';
+
 function Navigation(props) {
 
     return (
-        <nav>
-            <ul>
-                <li><NavLink to="/" activeClass="active">Home</NavLink></li>
-                <li><NavLink to="/users" activeClass="active">Users</NavLink></li>
-                <li>{props.needLogin 
-                 ? <NavLink to="/login" activeClass="active">Login</NavLink> 
-                 : <LogoutButton/>} </li>
-                
-            </ul>
+        <nav className="nav-main">
+            <div className="navigation-container">
+                <div className="nav-button">
+                    <div className="nav-icon"><i className="fas fa-plus"></i></div>
+                    <NavLink to="/campaign/create" activeClassName="active"><Button>START A CAMPAIGN</Button></NavLink>
+                </div>
+                <div className="logo"><NavLink to={`/`}>
```

Comment:
No need for curly-boiz!
```suggestion
                <div className="logo"><NavLink to="/">
```